strapi/strapi
# fix(admin): adjust onClick of create webhook button in empty state

### What does it do?

Adjust onClick props of "create new webhook" button of Settings -> Webhook page to make it works with `goTo` helper.

### Why is it needed?

It's needed because otherwise the button doesn't redirect the user to the webhook creation page.

### How to test it?

1. Navigate to "Settings" -> "Webhook"
2. Ensure to have no webhooks in your database
3. Click on "create new webhook" button diplayed in the content area of the page
4. Notice that you are now redirected to the creation page of a webhook

### Related issue(s)/PR(s)

Related to this issue #19257
